specification/src/main/asciidoc/platform/RelatedDocuments.adoc

```diff
@@ -220,39 +156,38 @@ instance and is independent of the number of Java virtual machines,
 operating system processes, or host machines used by the product
 instance.
 
-[.footnoteNumber]# 3.# [[a3650]]For a JSP page, this
-requirement applies to the _service_ method of the equivalent JSP page
+[.footnoteNumber]# 3.# [[a3650]]For a Jakarta Server Pages page, this
+requirement applies to the _service_ method of the equivalent Jakarta Server Pages page
 Implementation Class.
 
-[.footnoteNumber]# 4.# [[a3651]]Note that Java EE
-Managed Beans are required to have access to the JNDI naming environment
+[.footnoteNumber]# 4.# [[a3651]]Note that Jakarta Managed Beans are required to have access to the JNDI naming environment
 of their calling component.
 
 [.footnoteNumber]# 5.# [[a3652]]Note that the use of
 interceptors defined by means of the Interceptors annotation is
-supported in the absence of CDI for EJB and Managed Bean components.
+supported in the absence of Jakarta™ Contexts and Dependency Injection for Jakarta™ Enterprise Beans and Jakarta Managed Bean components.
 
-[.footnoteNumber]# 6.# [[a3653]]See the JSF
-specification section “JSF Managed Classes and Java EE Annotations” for
+[.footnoteNumber]# 6.# [[a3653]]See the Jakarta™ Server Faces
+specification section Jakarta™ Server Faces Managed Classes and Jakarta Annotations” for
 a list of these managed classes.
 
-[.footnoteNumber]# 7.# [[a3654]]Resource and CDI
-injection is supported only in JAX-RS components managed by CDI.
+[.footnoteNumber]# 7.# [[a3654]]Resource and Jakarta™ Contexts and Dependency Injection
```

Comment:
I would suggest we generalize this a bit...  "Resource, context, and dependency injection is supported..."